Signal/src/ViewControllers/ConversationView/Cells/MediaGalleryCellView.swift

```diff
@@ -142,11 +146,39 @@ public class MediaGalleryCellView: UIStackView {
                 moreLabel.autoCenterInSuperview()
             }
         }
+
+        for itemView in itemViews {
+            guard !moreItemViews.contains(itemView) else {
```

Comment:
TIOLI: Since we're skipping everything that's not in `moreItemViews`, would it make more sense to just iterate over the `moreItemViews`?

`for itemView in moreItemViews {`

Reply:
It's the opposite.  We're skipping everything that's _IN_ `moreItemViews`.